WordPress/Classes/ViewRelated/Blog/Settings/SiteSettingsViewModel.swift

```diff
@@ -40,6 +77,40 @@ final class SiteSettingsViewModel: ObservableObject {
         })
     }
 
+    // MARK: - Helpers (Timezone)
+
+    private func startTimezoneObserver() {
+        self.timezoneObserver = TimeZoneObserver { [weak self] _, newState in
```

Comment:
Moved this code from the old `SiteSettingsViewController` with almost no changes.